make_dashboards/templates/portal/index.html

```diff
@@ -117,13 +114,9 @@
             }
         });
         
-        // Header clock
-        updateClock();
-        
-        // Cron job runs on the hour, every hour. Get the web page to reload at 5 past the next hour
-        var reloadDelay = moment().add(1, 'hours').startOf('hour').add(5, 'minutes').diff();
-        setTimeout(function(){ location.reload(); }, reloadDelay );
```

Comment:
So, I guess you changed how the page is updated? I'm to lazy to search through the rest of the code / Slack history  to figure out how.